### ISSUE TITLE
This mod for "life" mod

### ISSUE THREAD

**Issue by tekniks58** [sethduda/AdvancedTowing#23]
Hello,

I going to you for a question, it's possible to recreate ou modify this mod for a "life" mod, a set this for class players ( for exemple policeman ).

This is a really great mod and very RP.

Thanks in advance for your help!

**Comment by sethduda** [comment]
Do you know the actual class names of the units? (E.g. B_soldier_repair_F)

**Comment by tekniks58** [comment]
No but it's just to set this for the work (e.g. policeman ) don't need "B_soldier_repair_F"
and i just want a compatibilities for altis life mod because i don't want set for only work, i want to anyone use tow vehicules but your mod does not function in altis life mod.

But in the futur maybe use the function for set this for only work.

**Comment by sethduda** [comment]
I'm confused - does this addon not work on life? or does it work, but you want to restrict the ability to tow to only cops?

**Comment by tekniks58** [comment]
yes i want to restrict the ability to tow to only cops or repairman or another work

**Comment by sethduda** [comment]
Do you know how to identify via scripting if a player is a copy or repairman?

**Comment by Amnezyk** [comment]
hi ! Nice script ! i install it only for Independent player (medic and repair man on my server), so i add this a the beggining of the scrypt 

 if(playerSide == independent) then { 

 and at the end   

   };

and it stop working :/ But if i remove that, the scrypt is ok. Any idea ? thank you :) (Sorry bad english, french baguette :D)

**Comment by sethduda** [comment]
That won't work. Try add adding the playerSide check in the if statement on line 789.

e.g.

if(!isNull player && isPlayer player && playerSide == independent) then {

**Comment by Amnezyk** [comment]
Thank you, i try and i come back to say if it's work !

**Comment by Amnezyk** [comment]
Work ! Thank you :)
Better script than Igiload !
